lnd.go

```diff
@@ -231,13 +233,22 @@ func lndMain() error {
 	}
 	idPrivKey.Curve = btcec.S256()
 
+	// Parse node color from configuration
+	hexColor, err := colorful.Hex(cfg.Color)
```

Comment:
Was pointed out that this parsing would be easy to do without importing a library, since it's on very specific format :) Good to not introduce unnecessary dependencies!